MoMA.Analyzer/Methods/Method.cs

```diff
@@ -151,28 +151,48 @@ public string ToStringWithClass ()
 		
 		private string ConvertType (string type)
 		{
-			switch (type) {
+            switch (type) {

```

Comment:
The original file is badly formatted. I can switch to using tabs, but then there are a lot of changes.